### PR TITLE
Update navbars example to include a centered nav with utilities

### PR DESCRIPTION
**Pull request by mdo**
Addresses #36486.

### CODE REVIEW

**Comment by SamuelMiller** [comment]
In the above example, the entire navbar collapses. I think there may be a simpler approach to getting a truly centered Navbar with left and right align-items. See my reply, which includes a Codepen example of the basic concept: https://github.com/twbs/bootstrap/discussions/36486#discussioncomment-2903439). 

Also, I would follow the example navbarTogglerDemo02 where only the menu and right-align item collapses, not the navbar-brand: https://getbootstrap.com/docs/5.2/components/navbar/#content

File: site/content/docs/5.2/examples/navbars/index.html [hunk 1]
```diff
@@ -400,6 +400,40 @@
       </div>
     </nav>
 
+    <nav class="navbar navbar-expand-lg bg-light rounded" aria-label="Twelfth navbar example">
```
Comment (julien-deramond) [inline site/content/docs/5.2/examples/navbars/index.html:403]:
```suggestion
    <nav class="navbar navbar-expand-lg bg-light rounded" aria-label="Thirtheenth navbar example">
```

**Review by julien-deramond — CHANGES_REQUESTED** [review]
This new example LGTM! Just a proposal to change the `aria-label`.
IDK if adding another example where only the menu and right-align item collapses and not the navbar-brand is relevant.

**Comment by SamuelMiller** [comment]
> Also, see my codepen example for a potentially different approach to center the menu, using flex-1 on both left and right items only using the BS class "col" without specifying a number suffix. [#36486 (comment)](https://github.com/twbs/bootstrap/discussions/36486#discussioncomment-2903439)

I have provided a Codepen of this approach applied to a Bootstrap navbar. See comment, https://github.com/twbs/bootstrap/discussions/36486#discussioncomment-2917883

**Pull request merged**